File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        std::vector<int>memory;
        int k = 0;
        for(int i = 0; i < nums.size(); i++){
            if(nums[i] != 1000000){
                if(std::find(memory.begin(), memory.end(), nums[i]) != memory.end()){
                    nums[i] = 1000000;
                }
                else{
                    memory.push_back(nums[i]);
                    k++;
                }
            }
        }
        sort(nums.begin(), nums.end());
        return k;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 0
        j = 0

        while(i < len(nums)):
            while(j < len(nums)):
                if nums[i] == nums[j] and i != j:
                    nums.pop(j)
                else:
                    j += 1
            j = 0
            i += 1

        return(len(nums)) 